app/models/concerns/scheme_definition_fetchable.rb

```diff
@@ -35,6 +31,20 @@ def resolve_context
 
   private
 
+  def temp_file uri
+    file = Tempfile.new(["tmpschemfile", infer_extension(uri)])
+    file.write(fetch_content(uri).force_encoding("UTF-8"))
+    file.rewind
+    file
+  end
+
+  def fetch_content uri
```

Comment:
Hmm, strangely enough, this addition made the tests ~60% slower on my machine (8m vs 5m). I guess we can investigate this regression later.

Reply:
Agree. I'll create an issue for this then.

Reply:
Issue created [#154](https://github.com/t3-innovation-network/desm/issues/154).